README.md

```diff
@@ -63,3 +63,39 @@ Override these rules in `tslint.json`:
 }
 ```
 
+## Inline Linting in Text Editors
```

Comment:
Is this branch based on your other branch, or vice-versa? Either way, probably want to try to not show the changes in two different PRs.